### PR TITLE
Charlesthebird/yarn audit fix

### PR DESCRIPTION
**Pull request by Charlesthebird**
This updates package versions so that yarn audit has 0 vulnerabilities now.

Resolves: https://github.com/solo-io/gloo-mesh-enterprise/issues/15008

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ WORKDIR /app/projects/ui
 
 # Install global dependencies and set up for runtime
 RUN apt-get update && apt-get install -y build-essential
-RUN yarn global add vite@4.2.3
+RUN yarn global add vite@4.5.2
```
Comment (Worfle) [inline Dockerfile:18]: curious about this line.  What are we bringing vite in for?

Comment (Charlesthebird) [reply Dockerfile:18]: Vite is installed globally here to serve it with `vite preview --port 4000 --host`.

But I think this was leftover from a few versions of the dockerfile ago - changing it to `vite preview --port 4000 --host` and seeing if we can take out this line so we don't need to upgrade the version.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:16.14.2 
+FROM node:16.14.2
```
Comment (Charlesthebird) [inline Dockerfile:1]: Looking at this file again I noticed I missed the upgrade here.

File: Dockerfile (hunk 1 above)
Comment (Charlesthebird) [reply Dockerfile]: Just built it with the new dockerfile changes and the image loaded correctly 🎉

**Review by toreysoloio — APPROVED** [review]

**Pull request merged**